lighthouse-core/report/html/report-styles.css

```diff
@@ -1139,6 +1142,16 @@
   will-change: opacity; /* Only using for layer promotion */
 }
 
+/* Category header gauges shouldn't be links. */
+.lh-category-wrapper .lh-gauge__wrapper {
+  pointer-events: none;
+  cursor: default;
+}
+.lh-gauge__label-info-icon {
+  pointer-events: fill;
+  cursor: help;
```

Comment:
feels a tad odd to me to flip the cursor since there's no actual click action here, is there a broader UX strategy here? I would love to learn more about these kinds of things :)

Reply:
honestly, I had the bring the cursor back with CSS so I just picked one. Since there's no click action to prompt perhaps it should just remain a normal cursor. IDK.

@yuinchen what do you think of this?
https://lighthouse-33lan68g9.now.sh/#accessibility (hover over `i`)

Reply:
@Hoten I suggest shifting the position 4px downward and to the right.

I applied `transform: translate3d(4px,4px,0)` to the info icon and it looks like this.

![Screen Shot 2019-05-30 at 10 22 10 AM](https://user-images.githubusercontent.com/406636/58650949-df1e0c00-82c4-11e9-8031-ab04273fa68d.png)